sdk/typescript/test/e2e/utils/setup.ts

```diff
@@ -68,9 +70,15 @@ export function getClient(url = DEFAULT_FULLNODE_URL): SuiClient {
 export async function setup(options: { graphQLURL?: string; rpcURL?: string } = {}) {
 	const keypair = Ed25519Keypair.generate();
 	const address = keypair.getPublicKey().toSuiAddress();
+	setupClientConfig();
 	return setupWithFundedAddress(keypair, address, options);
 }
 
+export async function setupClientConfig() {
```

Comment:
this is declared as an async function, but doesn't do anything async, and isn't awaited above, which could result in unhandled errors